src/cmd/compress_persistence.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2021 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <iostream>
+
+#include <boost/program_options.hpp>
+#include "depends/libDatabase/LevelDB.h"
+#include "libUtils/FileSystem.h"
+
+/// Should be run with working directory where folder "persistence" consisting
+/// of the individual dbs exists.
+
+enum class STATUS {
```

Comment:
I think (as opposed to my previous comment), it'd be nicer to just make it a regular enum (i.e. remove the 'class').
This way we can skip the ugly explicit cast to ints below (static_cast<int>(...)).